plugins/filter/ipaddr.py

```diff
@@ -548,6 +548,11 @@ def ipaddr(value, query="", version=False, alias="ipaddr"):
 
     vtype = None
 
+    if not value:
+        return False
+    elif value is True:
+        return False
+
```

Comment:
The point of the previous PR was to avoid returning False on an empty list, so that list operations can be chained. Rather than return this check, just replace the later `AnsibleFilterError` on line 562 with `return False`

Reply:
The current behaviour seems more correct IMO, but since this results in breaking change it is more appropriate in a major release of the collection.